pytorch_lightning/utilities/debugging.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+
+class InternalDebugger(object):
```

Comment:
I would rather add this in sepearte PR

Reply:
it's needed for these tests....

Reply:
Is this the debugger you were talking about with me?
I was working on this, is this now finished?